packages/@vuepress/shared-utils/src/getPermalink.ts

```diff
@@ -12,6 +12,37 @@ interface PermalinkOption {
 function removeLeadingSlash (path: string) {
   return path.replace(/^\//, '')
 }
+
+function toUtcTime (date: string | Date): number {
+  let year = 1970
+  let month = 0
+  let day = 1
+
+  if (typeof date === 'string') {
+    const [
+      yearStr,
+      monthStr,
+      dayStr
+    ] = date.split('-')
+
+    year = parseInt(yearStr, 10)
+    month = parseInt(monthStr, 10) - 1
+    day = parseInt(dayStr, 10)
+  } else if (date instanceof Date) {
```

Comment:
Here might be the problem. `date` is already in UTC time. There's no need to retrieve year, month, day and reassemble them.

Reply:
Makes sense. I've updated to do just that.